var/spack/repos/builtin/packages/thrift/package.py

```diff
@@ -89,6 +90,8 @@ def install(self, spec, prefix):
         options.append("--with-go=%s" % ("yes" if "+go" in spec else "no"))
         options.append("--with-lua=%s" % ("yes" if "+lua" in spec else "no"))
         options.append("--with-php=%s" % ("yes" if "+php" in spec else "no"))
+        options.append("--with-kotlin=%s" % ("yes" if "+kotlin" in spec else "no"))
+        options.append("--with-ruby=%s" % ("yes" if "+ruby" in spec else "no"))
```

Comment:
These need corresponding `variant()` directives.

Reply:
I chose not to build these libs instead. Someone can do it in the future if needed. Is that ok?

Reply:
Fine by me.